src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.qna.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Answers {
+    private final List<Answer> answers;
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public static Answers from(List<Answer> answers) {
+        return new Answers(answers);
+    }
+
+    public void add(Answer answer) {
+        this.answers.add(answer);
+    }
+
+    public boolean isNotOnlyWriter(NsUser loginUser) {
+        return !answers.stream().allMatch(answer -> answer.isOwner(loginUser));
```

Comment:
객체지향 생활 체조 원칙의 `규칙 4: 한 줄에 점을 하나만 찍는다.`를 적용해보면 어떨까요?
```suggestion
        return !answers.stream()
            .allMatch(answer -> answer.isOwner(loginUser));
```